pkgs/by-name/dc/dcfldd/package.nix

```diff
@@ -4,15 +4,23 @@
   fetchurl,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "dcfldd";
   version = "1.3.4-1";
 
   src = fetchurl {
-    url = "mirror://sourceforge/dcfldd/dcfldd-${version}.tar.gz";
+    url = "mirror://sourceforge/dcfldd/dcfldd-${finalAttrs.version}.tar.gz";
```

Comment:
Let’s switch to the https://github.com/resurrecting-open-source-projects/dcfldd fork, as used by Debian, which seems to have fixed these issues (and presumably many others).